tfhe/src/keycache/mod.rs

```diff
@@ -59,7 +61,7 @@ pub mod utils {
         }
     );
 
-    pub(crate) use named_params_impl;
+    pub use named_params_impl;
```

Comment:
I don't think this work as expected for cross-crate imports

Reply:
then we'll see when we get the issue